### ISSUE TITLE
Installation request

### ISSUE THREAD

**Issue by alysawyer** [mikeizbicki/modulus-magnus-linguae#75]
Hi Mike, was wondering if you could download some version of conda/miniconda/micromamba onto the Lambda server as I think it would be helpful for installing the dependancies associated with latin bert (preferably micromamba). I was trying to do it without but I think that was unideal.

**Comment by mikeizbicki** [comment]
I don't believe you need root access to install micromamba.  I've never used it before, but I just followed the instructions at <https://mamba.readthedocs.io/en/latest/micromamba-installation.html#install-script> with a non-root user and everything seemed to work fine for me.
